### ISSUE TITLE
How to disable seeking on media response?

### ISSUE THREAD

**Issue by shivan** [actions-on-google/assistant-conversation-nodejs#17]
How can I disable seeking on media responses? So the user cannot seek forward/backwards in the current playing track?

This is how I currently do it:

```
conv.add(new Media({
        mediaObjects: [{
        name: 'Trance Track 1',
        description: 'Media description',
        url: 'https://my-url.de/Trance1.mp3',
        image: {
          large: new Image({
            url: 'https://somewhere/whateverimage.jpg',
            alt: 'my image here'
          })
        }
    }],
        mediaType: MediaType.Audio,
        optionalMediaControls: [OptionalMediaControl.Paused, OptionalMediaControl.Stopped],
        startOffset: '0s'
    }));
```

I didn't find anything in the documentations on https://developers.google.com/assistant/conversational/prompts-media

Also didn't find anything on stackoverflow and didn't get any answer on stackoverflow, yet:
https://stackoverflow.com/questions/65039702/how-to-disable-manual-seeking-on-google-actions-assistant-conversations-media 

So my last option is to ask here.

**Comment by Fleker** [comment]
Is there a strict need for the user to be unable to seek? Potentially you can use a livestream of an mp3 rather than a static media file, which would make the content play as if it was live.

**Comment by shivan** [comment]
@Fleker, yes, there is a strict need, as it will be an action for a radio station but the files are static media files.

**Comment by shivan** [comment]
Any news here? Or any other suggestions than streaming?
Because seeking by code at the start should be possible, but not by the user. This is because it simulates radio streaming and should start at the same offset where the current "stream" is.

**Comment by Fleker** [comment]
From my understanding sending a standard media file will always show seeking.

**Comment by shivan** [comment]
But how does tunein and the others solve this - do all a "real" stream? For example SWR1 - there is no "live" icon like on tunein so is this still a stream?

We would really need to disable seeking, as this is a licensing limitation.

**Comment by Fleker** [comment]
TuneIn to my knowledge actually streams radio live.

**Comment by shivan** [comment]
Mhm... what can I do? As far as I know, there is no Developer Support contact mail address.

**Comment by Fleker** [comment]
What developer support are you looking for?

**Comment by shivan** [comment]
Like amazon is providing direct support for alexa skill developers in a ticketing system. Which such a support I was able to solve several problems with alexa already. For example devices behaving different or other problems.

Because without being able to disable seeking we cannot use google assistant. :-(

**Comment by Fleker** [comment]
I've filed a ticket internally.

**Comment by shivan** [comment]
Any news here?

**Comment by shivan** [comment]
Additionally I would need to disallow "next" (and "previous"). As skipping to the next track must be prevented, too.
(later I should be able to enable track skipping but with a very complex rule set)

**Comment by shivan** [comment]
@Fleker do you have any status update on that?

**Comment by Fleker** [comment]
I don't have an update to share.

**Comment by shivan** [comment]
Hello @Fleker , and now? How could we get forward here?

Streaming isn't an alternative, as it has two problems:
1) Metadata for changing tracks will not be shown
2) the player uses "Content-Length" as stream length, this cannot be disabled. 

It would be nice to hear from you.
